app/src/main/res/layout/recently_closed_nav_item.xml

```diff
@@ -49,7 +49,8 @@
         android:id="@+id/recently_closed_tabs_description"
         android:layout_width="0dp"
         android:layout_height="wrap_content"
-        android:layout_marginStart="12dp"
+        android:layout_marginStart="24dp"
+        android:layout_marginTop="2dp"
```

Comment:
You will want to update the text size for the header and description here as well and consider dropping the margin top as discussed above.

Reply:
done